front/pages/api/stripe/webhook.ts

```diff
@@ -325,4 +386,90 @@ async function handler(
   }
 }
 
+async function getAdminEmailsForSubscription(stripeSubscriptionId: string) {
```

Comment:
Could we move this logic and the email to another file? One where we would store all the automatic communication to admins?

Reply:
IMHO technical sorting VS functional sorting (e.g. all functions that do emails there VS all functions related to the Jacky feature ther) are both fine. Maybe tech sorting would be better here. But then would it be worth the time to refactor? Started to overthink it 🤔 => which made it moot so just did it 👍 (but here the ratio effort / gain was less obvious TBH)